### DISCUSSION TITLE
IGDB Alternative

### DISCUSSION THREAD

**Discussion by Emporea** [rommapp/romm#641] [Q&A]
IGDB requires Twitch, which is owned by Amazon. To access the API, Twitch requires phone number verification.
Therefore, I am seeking community input on this privacy-sensitive topic. I have been unable to find reliable community-driven open-source databases. Why is this the case? Does anyone know of any alternatives?

This project seems promising, but its heavy reliance on third-party proprietary services may prevent some users from using it.

**Comment by zurdi15** [comment]
Hi! we already have an opened [issue](https://github.com/zurdi15/romm/issues/212) with a lot of information about alternatives and pros/cons of each one.
Said that, you can expect to have more than one source for sure :)
